doc/github_actions.rst

```diff
@@ -0,0 +1,39 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration. If specified, a broken link will be created.
+
+Command line
+------------
+
+To use `universum` to check-out current repository (the repository that contains the GitHub Action workflow) is
+necessary to use `GitHub Environment variables
+<https://docs.github.com/en/actions/learn-github-actions/environment-variables>`_:
+::
+    python -m universum --vcs-type=git --git-repo "${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}" --git-refspec "${GITHUB_REF_NAME}"
+
+`Universum` also supports reports to review systems from GitHub Actions. Link to build are created using environment
```

Comment:
1. 'code review systems'
2. supports reporting to, not reports to
3. either 'links are created' or 'link is created'
4. I'm not so sure the clarification on 'links to build' is needed at all. Can we just mention that '--report-to-review' option is fully supported and doesn't require any further parameters passed to build?